### DISCUSSION TITLE
preproc_batch_tops.csh data.in dem.grd 2 &> pbt_mode2.log &

### DISCUSSION THREAD

**Discussion by wajidraki** [gmtsar/gmtsar#1010] [General]
When i run this commend "preproc_batch_tops.csh data.in dem.grd 2 &> pbt_mode2.log &"  i get  below error in log file .how i resolved this issue? even i use data provide by gmtsar as a sample task

*** buffer overflow detected ***: terminated
Aborted (core dumped)
 can't open %s for reading S1_20150427_134414_F1.SLC 
mv: cannot stat 'S1_20150427_ALL_F1.PRMresamp': No such file or directory
mv: cannot stat 'S1_20150427_ALL_F1.SLCresamp': No such file or directory
Successfully opened junk1 
[ERROR]:hermite interpolation point outside of data constraints, 10071834.261444 10071851.919000 10071867.919000
cat: junk2: No such file or directory
rm: cannot remove 'junk2': No such file or directory
using command line
......master LED file S1_20150403_134413_F1.LED 
.........aligned LED file S1_20150121_134413_F1.LED 
[ERROR]:hermite interpolation point outside of data constraints, 1777466.621987 1777450.259000 1777466.259000
using command line
......master LED file S1_20150403_134413_F1.LED 
.........aligned LED file S1_20150121_134413_F1.LED 
[ERROR]:hermite interpolation point outside of data constraints, 1777466.621987 1777450.259000 1777466.259000
using command line
......master LED file S1_20150403_134413_F1.LED 
.........aligned LED file S1_20150310_134412_F1.LED 
[ERROR]:hermite interpolation point outside of data constraints, 5924666.291033 5924649.910000 5924665.910000
using command line
......master LED file S1_20150403_134413_F1.LED 
.........aligned LED file S1_20150310_134412_F1.LED 
[ERROR]:hermite interpolation point outside of data constraints, 5924666.291033 5924649.910000 5924665.910000
using command line
......master LED file S1_20150403_134413_F1.LED 
.........aligned LED file S1_20150403_134413_F1.LED 
using command line
......master LED file S1_20150403_134413_F1.LED 
.........aligned LED file S1_20150403_134413_F1.LED 
using command line
......master LED file S1_20150403_134413_F1.LED 
.........aligned LED file S1_20150427_134414_F1.LED 
[ERROR]:hermite interpolation point outside of data constraints, 10071868.071505 10071851.919000 10071867.919000
using command line
......master LED file S1_20150403_134413_F1.LED 
.........aligned LED file S1_20150427_134414_F1.LED 
[ERROR]:hermite interpolation point outside of data constraints, 10071868.071505 10071851.919000 10071867.919000

**Comment by Xiaohua-Eric-Xu** [comment]
Seems answered here. https://github.com/gmtsar/gmtsar/issues/1002#issuecomment-2486759400

**Reply by wajidraki** [reply]
Thanks
